### PR TITLE
blasphemer rebalance, fixes #572

### LINKED ISSUE #572: Blasphemer Rebalance
When drawn add four Smites to your hand and exhaust.

Upgrade: When drawn add six Smites to your hand and exhaust.

### PR DESCRIPTION
**Pull request by acourrau**

### CODE REVIEW

File: src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java [hunk 1]
```diff
@@ -19,8 +19,8 @@
     public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
 
     private static final int COST = COST_UNPLAYABLE;
-    private static final int SMITE_AMOUNT = 5;
-    private static final int UPGRADE_SMITE_AMOUNT = 1;
+    private static final int SMITE_AMOUNT = 4;
+    private static final int UPGRADE_SMITE_AMOUNT = 2;
```
Comment (wang429) [inline src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java:23]: Do we know if he wanted 4(5) or 4(6)?

**Review by wang429 — CHANGES_REQUESTED** [review]
Thanks for the contribution! This change affects things a player would see. Could you update CHANGELOG.md too?

File: src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java (hunk 1 above)
Comment (acourrau) [reply src/main/java/stsjorbsmod/cards/cull/deckoftrials/Blasphemer.java:23]: He only mentioned that 5 should come down to 4, so I assumed he still wanted Blasphemer+ to be 6 still. Probably should clarify.

**Review by wang429 — APPROVED** [review]
Looks good to me! Thanks for the contribution.

**Pull request merged**